source/channels/create-channels.rst

```diff
@@ -43,11 +43,11 @@ Anyone can create public channels, private channels, direct messages, and group
       
   .. tab:: Mobile app
    
-    **To create a public or private channel in the mobile app V2**
+    **To create a public or private channel in the mobile app v2**
```

Comment:
We're going to avoid mentioning v2 specifically in our mobile app documentation since the mobile app v2 is still in beta.